cbits/fpstring.c

```diff
@@ -73,7 +73,7 @@ unsigned char fps_minimum(unsigned char *p, unsigned long  len) {
 }
 
 /* count the number of occurences of a char in a string */
-unsigned long fps_count(unsigned char *p, unsigned long len, unsigned char w) {
+unsigned long fps_count(unsigned char *p, size_t len, unsigned char w) {
```

Comment:
```suggestion
size_t fps_count(unsigned char *p, size_t len, unsigned char w) {
```
And also in the declartion of `c` below.